src/main/resources/assets/waila/lang/en_us.json

```diff
@@ -1,80 +1,80 @@
-  "config.waila.hide_from_debug": "Hide from Debug",
-  "config.waila.hide_from_debug_desc": "Hide the tooltip while the debug menu is open",
-  "config.waila.display_item": "Display Item",
-  "config.waila.display_item_desc": "Display the item gotten from the block",
-  "config.waila.tts": "Enable TTS",
-  "config.waila.tts_desc": "Read out block and entity names with the system's text to speech processor",
+  "config.waila.general": "Général",
+  "config.waila.display_tooltip": "Afficher l'infobulle",
+  "config.waila.display_tooltip_desc": "Récupérer des données et rendre l'infobulle",
+  "config.waila.display_fluids": "Afficher les fluides",
+  "config.waila.display_fluids_desc": "Essayer d'afficher les fluides",
+  "config.waila.sneaky_details": "Détails",
+  "config.waila.sneaky_details_desc": "Le corps du texte est caché sauf si vous êtes accroupi",
+  "config.waila.display_mode": "Mode affichage",
+  "config.waila.display_mode_desc": "Comportement de l'infobulle\nBasculer : appuyer sur le raccourci pour l'activer ou le désactiver\nMaintenir : la touche doit être maintenue",
+  "config.waila.display_mode_toggle": "Basculer",
+  "config.waila.display_mode_hold_key": "Maintenir",
+  "config.waila.hide_from_players": "Cacher de la liste des joueurs",
+  "config.waila.hide_from_players_desc": "Cacher l'infobulle lorsque la liste des joueurs est ouverte",
+  "config.waila.hide_from_debug": "Cacher du débug",
+  "config.waila.hide_from_debug_desc": "Cacher l'infobulle lorsque le menu F3 est ouvert",
+  "config.waila.display_item": "Afficher l'objet",
+  "config.waila.display_item_desc": "Afficher l'objet obtenu d'un bloc",
+  "config.waila.tts": "Activer le narrateur",
+  "config.waila.tts_desc": "Lire le nom des blocs et entités avec le système de narrateur",
 
   "config.waila.overlay": "Overlay",
-  "config.waila.overlay_pos_x": "Horizontal Ratio",
-  "config.waila.overlay_pos_y": "Vertical Ratio",
-  "config.waila.overlay_scale": "Display Scale",
+  "config.waila.overlay_pos_x": "Ratio horizontal",
+  "config.waila.overlay_pos_y": "Ratio vertical",
+  "config.waila.overlay_scale": "Taille d'affichage",
 
-  "config.waila.overlay_color": "Color",
+  "config.waila.overlay_color": "Couleur",
   "config.waila.overlay_alpha": "Alpha",
-  "config.waila.overlay_theme": "Theme",
+  "config.waila.overlay_theme": "Thème",
 
-  "config.waila.formatting": "Formatting",
-  "config.waila.format_mod_name": "Mod Name",
-  "config.waila.format_block_name": "Block Name",
-  "config.waila.format_fluid_name": "Fluid Name",
-  "config.waila.format_entity_name": "Entity Name",
-  "config.waila.format_registry_name": "Registry Name",
+  "config.waila.formatting": "Mise en forme",
+  "config.waila.format_mod_name": "Nom du mod",
+  "config.waila.format_block_name": "Nom du bloc",
+  "config.waila.format_fluid_name": "Nom du fluide",
+  "config.waila.format_entity_name": "Nom de l'entité",
+  "config.waila.format_registry_name": "Nom du registre",
```

Comment:
This is the entry's registry name as in the name in the registry. Just like your family name is not "nom de la famille", this should be "Nom **de** registre"